### PR TITLE
Logging upgrade

### LINKED ISSUE #984: Add logging subscription to the platform log
Allows logging from apps before the application has actually started from the platform

### PR DESCRIPTION
**Pull request by craig8**
# Description

Updates log manager to include logging to the platform.log as well as the simulation log.  The platform.log is where applications that haven't been started can log (i.e. the application wrapper python).  

Fixes #984 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Tested with the sample app and newest python code

### CODE REVIEW

**Comment by craig8** [comment]
Need to push a change to gridappsd-python and the sample-app for this to be fully tested, however this won't break any current functionality.

File: gov.pnnl.goss.gridappsd/src/gov/pnnl/goss/gridappsd/process/ProcessManagerImpl.java [hunk 1]
```diff
@@ -1,44 +1,55 @@
...
  * herein do not necessarily state or reflect those of the United States Government or any agency thereof.
- * 
- * PACIFIC NORTHWEST NATIONAL LABORATORY operated by BATTELLE for the 
+ *
+ * PACIFIC NORTHWEST NATIONAL LABORATORY operated by BATTELLE for the
  * UNITED STATES DEPARTMENT OF ENERGY under Contract DE-AC05-76RL01830
  ******************************************************************************/
 package gov.pnnl.goss.gridappsd.process;
 
+import java.util.Date;
```
Comment (poorva1209) [inline gov.pnnl.goss.gridappsd/src/gov/pnnl/goss/gridappsd/process/ProcessManagerImpl.java]: Moved import position?

File: gov.pnnl.goss.gridappsd/src/gov/pnnl/goss/gridappsd/process/ProcessManagerImpl.java [hunk 2]
```diff
@@ -77,42 +76,42 @@
  */
 @Component
 public class ProcessManagerImpl implements ProcessManager {
-	
+
```
Comment (poorva1209) [inline gov.pnnl.goss.gridappsd/src/gov/pnnl/goss/gridappsd/process/ProcessManagerImpl.java]: Lot of white space changes in this file ?

File: gov.pnnl.goss.gridappsd/src/gov/pnnl/goss/gridappsd/dto/LogMessage.java [hunk 3]
```diff
@@ -1,40 +1,40 @@
 /*******************************************************************************
  * Copyright (c) 2017, Battelle Memorial Institute All rights reserved.
- * Battelle Memorial Institute (hereinafter Battelle) hereby grants permission to any person or entity 
- * lawfully obtaining a copy of this software and associated documentation files (hereinafter the 
- * Software) to redistribute and use the Software in source and binary forms, with or without modification. 
- * Such person or entity may use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of 
+ * Battelle Memorial Institute (hereinafter Battelle) hereby grants permission to any person or entity
```
Comment (poorva1209) [inline gov.pnnl.goss.gridappsd/src/gov/pnnl/goss/gridappsd/dto/LogMessage.java]: White space changes in Copyright?

**Comment by craig8** [comment]
I used the.built to n eclipse code format in save thingy.  I think we all should.

**Pull request merged**